components/ProfileSubmissions.tsx

```diff
@@ -51,11 +55,28 @@ const ProfileSubmissions: React.FC<LessonChallengeProps> = ({ lessons }) => {
     const filterPassedChallenges = lesson.challenges.filter(
       eachChallenge => eachChallenge.challengeStatus === 'passed'
     )
+    const starsContainer = (
+      <p className={`lessonimage_progressbadge badge badge-pill badge-primary`}>
+        {lesson.starsReceived && lesson.starsReceived.length}
+        <span className="ml-1">
+          <Star size={15} fill="yellow" />
+        </span>
+      </p>
+    )
+    const starbadge =
+      lesson.starsReceived && lesson.starsReceived.length ? (
+        starsContainer
+      ) : (
+        <></>
+      )
```

Comment:
I think this is clearer than tertiary operation:

```
let starBadge = < />
if (lesson.starsReceived && lesson.starsReceived.length) {
  starBadge = (
    ...
  }
}
```

btw... `starBadge` (capitalize `B`)